kern/ept.c

```diff
@@ -750,7 +750,8 @@ static void ept_mmu_notifier_release(struct mmu_notifier *mn,
 }
 
 static const struct mmu_notifier_ops ept_mmu_notifier_ops = {
-	.invalidate_page	= ept_mmu_notifier_invalidate_page,
+	// TODO(alevy): I believe this needs to be converted to `invalidate_range`
+	//.invalidate_page	= ept_mmu_notifier_invalidate_page,
```

Comment:
invalidate page was removed. There is `invalidate_range` now, but it's not the same signature, and doesn't seem to have the same semantics.